### ISSUE TITLE
Untrusted message 

### ISSUE THREAD

**Issue by tigertail100** [EliasKotlyar/Xiaomi-Dafang-Hacks#580]
When opening the IP address in any browser (Windows 10) I get this connection is not secure and https is in red and a line through it
Regards
Ian

**Comment by jmtatsch** [comment]
Yes, because our http server uses a self- generated certificate which is not in the whitelist of any browser. If you want you can get a cert from let’s encrypt but that is much more complicated.
